discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -621,6 +621,18 @@ def wait_for_install(self, timeout=consts.CLUSTER_INSTALLATION_TIMEOUT):
             timeout=timeout,
         )
 
+    def _set_dns(self):
+        cluster = self.api_client.cluster_get(self.id)
+        cluster_name = cluster.name
+        base_domain = cluster.base_dns_domain
+        main_cidr = self.nodes.controller.get_machine_cidr()
+        nameserver_ip = str(IPNetwork(main_cidr).ip + 1)
+        fname = f"/etc/NetworkManager/dnsmasq.d/openshift-{cluster_name}.conf"
+        contents = f"server=/api.{cluster_name}.{base_domain}/{nameserver_ip}\n" + f"server=/.apps.{cluster_name}.{base_domain}/{nameserver_ip}\n"
```

Comment:
Using multi-line with `textwrap.dedent` would make it readable.

```suggestion
        from textwrap import dedent
        ....
        
        contents = dedent(f"""
        server=/api.{cluster_name}.{base_domain}/{nameserver_ip}
        server=/.apps.{cluster_name}.{base_domain}/{nameserver_ip}""")
```

Reply:
Is it more readable?

Reply:
Yes. You don't have one long line with strings concatenation nor new-line backslash.